src/gr11review/part2/Utility.java

```diff
@@ -8,7 +10,158 @@
+                intVowel1++;
+            }
+        }
+        // Check vowels in the word 2
+        for(int x = 0; x < strLine2.length(); x++){
+            if(strLine2.charAt(x) == 'a' || strLine2.charAt(x) == 'e' || strLine2.charAt(x) == 'i' || strLine2.charAt(x) == 'o' || strLine2.charAt(x) == 'u'){
+                intVowel2++;
+            }
+        }
+
+        // Compare the 2 vowel counts
+        if(intVowel2 > intVowel1){
+            strResult = strLine2; 
+        } else {
+            strResult = strLine1;
+        }
+        
+        intVowel1 = 0;
+        intVowel2 = 0;
+        strLine1 = text.readLine();
+        strLine2 = text.readLine();
+        }
+        text.close();
+    
+        // Return answer
+        return strResult;
+    }
+
+    /**
+     * Rearranged so that all the zeros are grouped at the start of the array
+     * @param nums How many numbers the array goes to 
+     * @return Return the answer with the zeros in the front 
+     * @author Parker Yang
+     */
+    public static int[] zeroFront(int[] nums){
+        // Declare variables 
+        int intPosition = 0;
+
+        // For loop for check the whole array 
+        for(int i = 0; i < nums.length; i++){
+            // If the number is 0, make its position in the array at the start
+            if(nums[i] == 0){
+                nums[i] = nums[intPosition];
+                nums[intPosition] = 0;
+                intPosition++;
+            }
+        }
+        // Return answer
+        return nums;
+    }
+
+    /**
+     * Repeats the number in an array till the max number
+     * @param n Max number the array goes up to 
+     * @return Return the max number the array goes to 
+     * @author Parker Yang
+     */
+    public static int[] seriesUp(int n){
+        // Declare variables
+        int intNumber = 0; 
+        int[] intArray = new int[n * (n + 1) / 2];
+
+        // For loop to add numbers to continue the array 
+        for(int i = 1; i <= n; i++){
+            for(int x = 1; x <= i; x++){
+                intArray[intNumber++] = x;
+            }
+        }
+        // Return answer
+        return intArray;
+    }
+
+    /**
+     * Takes a 2D array and reverses all of the content in the 2D array
+     * @param arr Takes the given array to reverse it 
+     * @return Return the reversed array 
+     * @author Parker Yang
+     */
+    public static int[][] reverse(int[][] arr){
+        int [][] intResult = new int[arr.length][arr[0].length];
+
+        // Declare variables 
+        int intMax1 = arr[0].length - 1;
+        int intMax2 = arr.length - 1;
+
+        // Loop to reverse the array
+        for(int intRow = 0; intRow <= intMax1; intRow++){
+            for(int intColumn = 0; intColumn <= intMax2; intColumn++){
+                intResult [intMax1 - intRow][intMax2 - intColumn] = arr[intRow][intColumn];
+            }
+        }
+
+        // Return answer
+        return intResult;
 
```

Comment:
There is an issue here in that the `}` for this `reverse` method is missing. Perhaps accidentally deleted on a merge.